README.md

```diff
@@ -35,13 +35,17 @@ For information regarding locally cached versions of Node.js on GitHub hosted ru
 
 ### Supported version syntax
 
-The `node-version` input supports the following values:
+The `node-version` input supports the [Semantic Versioning Specification](https://github.com/npm/node-semver), for example:
```

Comment:
Could you please change it to 
The `node-version` input supports the Semantic Versioning Specification, for more detailed examples please refer to the [documentation](https://github.com/npm/node-semver).

Reply:
n/p